joint_state_broadcaster/package.xml

```diff
@@ -6,7 +6,6 @@
   <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
   <maintainer email="denis@stogl.de">Denis Stogl</maintainer>
   <maintainer email="jordan.palacios@pal-robotics.com">Jordan Palacios</maintainer>
-  <maintainer email="karsten@openrobotics.org">Karsten Knese</maintainer>
```

Comment:
Let's change list of maintainers in a separate PR and in all packages at once.

Reply:
Yessir